presto-mysql/src/test/java/io/prestosql/plugin/mysql/TestMySqlDistributedQueries.java

```diff
@@ -156,5 +152,16 @@ protected boolean isColumnNameRejected(Exception exception, String columnName, b
         return Optional.of(dataMappingTestSetup);
     }
 
+    @Test
+    @Override
+    public void testDelete()
+    {
+        // Mysql connector currently does not support row-by-row delete
```

Comment:
Why?

- add a test with an equality condition on a bigint column -- should work
- add a test with a range condition on a bigint column -- should work
- add a test with an equality condition on a varchar column -- should not work, since filter is not fully removed on Presto side